projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -975,6 +975,11 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
         }
     }
 
+    public onInputGroupClick(event: MouseEvent, target?: HTMLElement) {
+        // TODO: should we call stopPropagation here?
```

Comment:
Don't see a reason to stop propagation of the click TBH

Reply:
resolved